instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ base.archivesName.set("opentelemetry-spring-boot")
 group = "io.opentelemetry.instrumentation"
 
 val versions: Map<String, String> by project
-val springBootVersion = versions["org.springframework.boot"]
+val springBootVersion = "2.7.18"
```

Comment:
I guess this version won't be updated any more?

Reply:
yes - any idea how to make it work?